ArduCopter/land_detector.cpp

```diff
@@ -76,7 +76,10 @@ void Copter::update_land_detector()
         // if we have a healthy rangefinder only allow landing detection below 2 meters
         bool rangefinder_check = (!rangefinder_alt_ok() || rangefinder_state.alt_cm_filt.get() < LAND_RANGEFINDER_MIN_ALT_CM);
 
-        if (motor_at_lower_limit && accel_stationary && descent_rate_low && rangefinder_check) {
+        // if we have weight on wheels (WoW) or ambiguous unknown. never no WoW
+        bool WoW_check = (landinggear.get_wow_state() == 1 || landinggear.get_wow_state() == -1) ? true : false;
```

Comment:
```suggestion
        bool WoW_check = (landinggear.get_wow_state() == 1 || landinggear.get_wow_state() == -1);
```

Reply:
Could you please explain a little bit what `const` means in this context? To me the value changes and the constant seems odd.

Reply:
`const` simply means the variable can't be modified past its initialisation.  That can help both the compiler and the reader reason about the code.

The ternary operator was just noise - you've got a boolean expression - the ternary operator construct here is a tautology.